### DISCUSSION TITLE
Cannot find package 'react-dom'

### DISCUSSION THREAD

**Discussion by GMKBabu7** [resend/react-email#1728] [Q&A] [answered]
Hi,

Can you please help me? I'm getting the issue below in NestJs.

Cannot find package 'react-dom'  imported from /usr/src/app/node_modules/@react-email/render/dist/node/index.js

Versions:
   "react-email": "^3.0.1",
       "@react-email/components": "^0.0.25",
    "@react-email/render": "^1.0.1",
        "react-dom": "18.2.0",
        
       
       import { renderAsync } from '@react-email/components';
               const htmlData = await renderAsync(
          templateFunctionName(payload)
        );
        
  Thanks,
  GMK Babu.

**Comment by gabrielmfern** [comment]
Might be too obvious, but did you run install after adding `react-dom`?

**Reply by GMKBabu7** [reply]
Hi gabrielmfern,

I'm still getting the error after installing react-dom.

**Reply by gabrielmfern** [reply]
Could you make a reproduction that I can test things on? Can't really help you otherwise.

**Reply by GMKBabu7** [reply, marked as answer]
Thanks, it's working after clearing the Yarn cache.
